spec.md

```diff
@@ -1318,6 +1369,12 @@ data that is read out of the read port is undefined.
 In all cases, if a memory location is written to by more than one port on the
 same cycle, the stored value is undefined.
 
+### Constant memory type
+
+A memory with a constant data-type represents a ROM and may not have 
+write-ports.  It is beyond the scope of this specification how ROMs are 
+initialized.
```

Comment:
Is it worth outlawing constant memories until we have a mechanism to initialize them?

Reply:
The spec doesn't give a way to initialize non-constant memories either.   I suspect in the near term, the existing defacto annotation or $readmem* intrinsic would work until all memories have a uniform initialization method available in the spec.

Reply:
I'm not sure that the `mem` construct is actually what we want to represent a ROM though. To me, it seems much more akin to an aggregate literal expression/construct, than a memory. In that regard, initialization of a memory is very different than a ROM, as the initialization of a ROM is not programmable and thus is synthesizable.